app/src/main/java/com/codefororlando/petadoption/persistence/mapper/AnimalMapper.kt

```diff
@@ -0,0 +1,33 @@
+package com.codefororlando.petadoption.persistence.mapper
+
+import com.codefororlando.petadoption.data.model.Animal
+import com.codefororlando.petadoption.persistence.model.AnimalEntity
+import javax.inject.Inject
+
+
+class AnimalMapper @Inject constructor(){
+
+    fun map(from: Animal): AnimalEntity {
```

Comment:
I would strongly suggest avoiding the function name of map in kotlin as it will be confusing to a reader. A more specific name would remove any confusion. `mapToAnimalEntity`